src/main/java/uk/gov/ons/census/casesvc/model/entity/Case.java

```diff
@@ -113,8 +113,7 @@
   @Column(nullable = false, columnDefinition = "BOOLEAN DEFAULT false")
   private boolean receiptReceived;
 
-  @Column(nullable = false, columnDefinition = "BOOLEAN DEFAULT false")
-  private boolean refusalReceived;
+  @Column private String refusalReceived;
```

Comment:
Again curious why we're not using the enum here?

Reply:
we absolutely **should** be using enums, otherwise the code will be horrible